.github/workflows/release-wave.yml

```diff
@@ -153,3 +153,78 @@ jobs:
         with:
           packages_dir: py/h2o_lightwave_web/dist
           password: ${{ secrets.PYPI_LIGTHWAVE_WEB_TOKEN }}
+
+  create-bundle:
+    name: Create Wave Bundle
+    runs-on: ubuntu-latest
+
+    outputs:
+      build-version: ${{ env.VERSION }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download H2O CLI
+        working-directory: ./py/apps/tour
+        run: mkdir -p .bin && curl -o .bin/h2o https://h2oai-cloud-release.s3.amazonaws.com/releases/ai/h2o/h2o-cloud/latest/cli/linux-amd64/h2o
+
+      - name: Change permissions
+        working-directory: ./py/apps/tour
+        run: chmod +x .bin/h2o
+
+      - name: Set version
+        working-directory: ./py/apps/tour
+        run: sed -i -r -e "s/\{\{VERSION\}\}/${{ env.VERSION }}/g" app.toml
+
+      - name: Get App Version
+        id: get-build-version
+        run: |
+          echo "VERSION=${{ env.VERSION }}" >> "$GITHUB_OUTPUT"
+
+      - name: Make the folder structure
```

Comment:
This introduces unwanted code duplication as this code can also be found in `make build-apps` target. Please refactor so that we have a single `make build-tour-app` target that can be reused here and also in `make build-apps`.

Reply:
Good Suggestion. Thanks @mturoci